packages/neo-one-website/src/components/docs/AdjacentLink.tsx

```diff
@@ -0,0 +1,42 @@
+import { Link } from '@neo-one/react-common';
+import * as React from 'react';
+import { Link as RouterLink } from 'react-static';
+import { as, Box, styled } from 'reakit';
+import { prop } from 'styled-tools';
+import { AdjacentInfo } from '../../utils';
+
+export interface Props {
+  readonly adjacent: AdjacentInfo;
+  readonly next: boolean;
+}
+
+const ArticleText = styled(Box)`
+  ${prop('theme.fonts.axiformaRegular')};
```

Comment:
Add a `prop('theme.fontStyles')` to make it explicit. Technically I think all font defaults to body1, but it's better to be explicit in this case - the default is more of a catch-all just in case we forget to be explicit somewhere.